src/ProjectSystem/Test/Microsoft.VisualStudio.ProjectSystem.FileSystemMirroring.Test.csproj

```diff
@@ -33,6 +33,13 @@
     <IntermediateOutputPath>$(BaseIntermediateOutputPath)\$(Configuration)\$(AssemblyName)\</IntermediateOutputPath>
     <OutputPath>$(BaseOutputPath)\$(Configuration)\</OutputPath>
   </PropertyGroup>
+  <ItemGroup Condition="'$(VisualStudioVersion)'=='14.0'" >
```

Comment:
Why not reference these directly from VS install folder? That way we won't have to keep checking the new ones in. And if a new VS version breaks things again, you'll find out during build.